cmd/habitat/community/consensus/raft/service.go

```diff
@@ -57,28 +57,36 @@ func (cs *ClusterService) Start() error {
 }
 
 // CreateCluster initializes a new Raft cluster, and bootstraps it with this nodes address
-func (cs *ClusterService) CreateCluster(communityID string) error {
+func (cs *ClusterService) CreateCluster(communityID string, initState []byte) (<-chan state.StateUpdate, error) {
```

Comment:
thoughts on making the state update channel part of the cluster service or cluster manager type instead of passing it around everywhere?